community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/LogicalPlan2PlanDescription.scala

```diff
@@ -43,7 +43,7 @@ object LogicalPlan2PlanDescription extends ((LogicalPlan, Map[LogicalPlan, Id])
         PlanDescriptionImpl(id = idMap(plan), "NodeIndexSeek", NoChildren, Seq(Index(label.name, propKey.name)), symbols)
 
       case NodeIndexUniqueSeek(IdName(id), label, propKey, value, arguments) =>
-        PlanDescriptionImpl(id = idMap(plan), "NodeIndexUniqueSeek", NoChildren, Seq(Index(label.name, propKey.name)), symbols)
+        PlanDescriptionImpl(id = idMap(plan), "NodeUniqueIndexSeek", NoChildren, Seq(Index(label.name, propKey.name)), symbols)
```

Comment:
This is the only place we are using this particular order,  and this is because we got it wrong in the interpreted runtime. I would like to fix that at some other point.